tests/integration/jinad/Dockerfiles/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7.6-slim
+FROM jinaai/jinad
```

Comment:
how does this image guarantee that `jina-core` is tested with the level of the PR

Reply:
[This](https://github.com/jina-ai/jina/blob/jinad-tests-branch/tests/integration/jinad/Dockerfiles/Dockerfile) pulls jinaai/jinad & just reinstalls jina from latest code, then restarts jinad